packages/js-sdk/tests/sandbox/host.test.ts

```diff
@@ -21,3 +24,36 @@ sandboxTest('ping server in sandbox', async ({ sandbox }) => {
     }
   }
 })
+
+// TODO: unskip when we have this new error handling deployed
+sandboxTest.skip('ping server in non-running sandbox', async () => {
+  const sbx = await Sandbox.create(template, { timeoutMs: 60_000 })
+
+  const cmd = await sbx.commands.run('python -m http.server 8000', {
+    background: true,
+  })
+
+  try {
+    await wait(10_000)
+
+    const host = sbx.getHost(8000)
+
+    const res = await fetch(`${isDebug ? 'http' : 'https'}://${host}`)
+
+    assert.equal(res.status, 200)
+
+    await sbx.kill()
+
+    const res2 = await fetch(`${isDebug ? 'http' : 'https'}://${host}`)
```

Comment:
We can use envd here, so we don't have to wait for the server to start, maybe we could do the same above.
```suggestion
  // Envd port
  const host = sbx.getHost(49983)

  const url = `${isDebug ? 'http' : 'https'}://${host}/health`
  const res = await fetch(url)

    assert.equal(res.status, 204)

    await sbx.kill()

    const res2 = await fetch(url)
```

Reply:
I don't think we would get the wanted `nginx` response with local envd

Reply:
You're right, skip this test if debug 🙏🏻